torchbox/careers
# Update Academy Page API Request

## [Link to Ticket On Codebase](https://projects.torchbox.com/projects/contentful-careers-page/tickets/113)

This ticket updates the GraphQL query to pull all the revised data from the new academy page designs (see link in ticket).

### How to Test

The preview site should load the academy page at the below link and display the hero subtitle and description. The build should compile without errors.

[Link to preview site page with component]()

### MR Checklist
